apps/meteor/server/startup/migrations/v314.ts

```diff
@@ -0,0 +1,32 @@
+import { Settings } from '@rocket.chat/models';
+
+import { addMigration } from '../../lib/migrations';
+
+addMigration({
+	version: 314,
+	name: 'Update default behavior of E2EE settings, to not allow un-encrypted messages by default and allow files encryption by default.',
+	async up() {
+		await Settings.updateOne(
+			{
+				_id: 'E2E_Allow_Unencrypted_Messages',
+			},
+			{
+				$set: {
+					value: false,
```

Comment:
Why not changing just the packageValue? So if someone has it as `true` and likes it being `true`, we don't break his workflow

Reply:
This should be said to the customer support team too, so if they receive comments from customers they know what to do 👀